gen/builders/builder.go

```diff
@@ -55,6 +55,7 @@ type Builder struct {
 	Actors    *Actors
 	Assert    *Asserter
 	Messages  *Messages
+	Traces    []string
```

Comment:
Suggestion: accumulate under an `[]*ExecutionTrace` slice here, in case we want to provide asserts that operate on the traces going forward. Then serialize in batch when spitting out the vector JSON.

Reply:
preference on using the lotus type for the internal structure versus maintaining a format in this repo for this?

Reply:
deferred serialization